crates/solidity/outputs/cargo/tests/src/bindings_output/runner.rs

```diff
@@ -73,7 +74,7 @@ pub fn run(group_name: &str, test_name: &str) -> Result<()> {
             // Don't run this in CI, since the graph outputs are not committed
```

Comment:
nit suggestion: only generating these mermaid graphs if a certain env variable is set, as we don't have to pay this cost in 99% of the cases.
Additionally, if you're planning on writing it in `/target` folder instead, I suggest just using `std::fs::write_file()` for the graph instead of `CodegenFileSystem::write_file()` as it performs a few extra operations that only apply to files we check into git.